bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -88,16 +88,16 @@
         <span class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
         <ul id="nav-main-menu">
           <li class="first about-item">
-            <a data-link-type="about" data-element-location="nav click" href="{{ url('mozorg.about') }}">{{_('About')}}</a>
+            <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{_('About')}}</a>
```

Comment:
`.manage.py test` just runs the python unit tests. See here for how to run the functional tests if you need to: http://bedrock.readthedocs.org/en/latest/testing.html